src/cpp/session/modules/SessionDataViewer.R

```diff
@@ -85,6 +87,15 @@
    vals
 })
 
+.rs.addFunction("formatDataColumnList", function(col, ...)
+{
+   formatted <- as.character(col)
+   large <- nchar(formatted) > 50L
```

Comment:
Is there a number / R options we could use here to control this? 50 characters might feel too small for some users.

Reply:
+1, would also suggest regardless you declare 50L as a constant instead of setting it in multiple places, which will make it harder to change

Reply:
There is `options()$width` but depending where the console is, it might be negatively correlated to the space there is in the data viewer ...

Reply:
Yeah I think perhaps you could make a new one, with 50 as the default, similar to how we have `data_viewer_max_columns` defined [here](https://github.com/rstudio/rstudio/blob/483ee0dbf1389e183e003c3a0bc1405c4c45aba9/src/cpp/session/resources/schema/user-prefs-schema.json#L1257), and [here ](https://github.com/rstudio/rstudio/blob/483ee0dbf1389e183e003c3a0bc1405c4c45aba9/src/cpp/session/prefs/UserPrefValues.cpp#L2421) and other places